test/integration/net_test.go

```diff
@@ -70,6 +70,10 @@ func TestNetworkPlugins(t *testing.T) {
 					t.Skipf("flannel is not yet compatible with Docker driver: iptables v1.8.3 (legacy): Couldn't load target `CNI-x': No such file or directory")
 				}
 
+				if ContainerRuntime() != "docker" && tc.name == "false" {
```

Comment:
@ilya-zuyev 
in case of containerd , should even allow users to specify the cni to be false?

I tried on my machine the coreDNS will not come up 
```
NAMESPACE     NAME                               READY   STATUS              RESTARTS   AGE
kube-system   coredns-74ff55c5b-qjj8r            0/1     ContainerCreating   0          58s
kube-system   etcd-minikube                      1/1     Running             0          70s
kube-system   kube-apiserver-minikube            1/1     Running             0          70s
kube-system   kube-controller-manager-minikube   1/1     Running             0          70s
kube-system   kube-proxy-6spkq                   1/1     Running             0          58s
kube-system   kube-scheduler-minikube            1/1     Running             0          70s
kube-system   storage-provisioner                1/1     Running             1          71s
```

I think the the right thing is, instead of skipping the test inside minikube test itself we should check if user passed `--cni=false` with non-docker runtime, we should Exit with usage that if you really want this you need to pass --force otherwise coredns needs this and wont come up

Reply:
and then for this test instead of skipping for non-docker we should check if it Exited with Usage Exit code or not